### DISCUSSION TITLE
3D kriging takes very long time to execute

### DISCUSSION THREAD

**Discussion by askar-INTERA** [GeoStat-Framework/PyKrige#224] [Q&A]
Hi I am doing 3d ordinary kriging using 50 points that yield the values that I want to interpolate over 448 points only. This takes forever to be executed and I am not sure why, can you please help.

HK = pd.read_csv ('HK_3D.csv')
PP = pd.read_csv ('PP_HK_3D.csv')
parameters={'sill': 15.0, 'range': 30, 'nugget': 0.0}
ok = OrdinaryKriging3D(PP['x'], PP['y'], PP['z'], PP['k'], variogram_model = "exponential", variogram_parameters=parameters, enable_plotting=True, verbose=True) 
k3d1, ss3d = ok.execute("grid", HK['x'], HK['y'], HK['z'])

**Comment by MuellerSeb** [comment]
That is strange. Have tried to do the same with GSTools?

I would also need some more information.
